tests/constants.py

```diff
@@ -0,0 +1,95 @@
+FAKE_NET_COMMAND = "The request will be processed at a domain controller for domain <DOMAIN>\
```

Comment:
two constant files ?
maybe you can merge them or rename them
same here, you can create an enum ;)

Reply:
Transform into fixture